frontend/src/components/explorer/mapViewer/SidebarDataPanels.vue

```diff
@@ -155,19 +155,16 @@ export default {
       selectedElementDataKeys: {
         metabolite: [
           { name: 'name' },
-          { name: 'model_name', display: 'Model&nbsp;name' },
           { name: 'formula' },
           { name: 'compartment' },
-          { name: 'synonyms', display: 'Synonym(s)' },
         ],
         gene: [
           { name: 'name', display: 'Gene&nbsp;name' },
-          { name: 'alternate_name', display: 'Alt&nbsp;name' },
-          { name: 'synonyms', display: 'Synonym(s)' },
```

Comment:
May I ask what was the rationale behind removing the synonyms?

Reply:
Good catch, it was causing problems and I meant to fix this actually but forgot about it.